ocs_ci/ocs/resources/cbt_metadata.py

```
@@ -0,0 +1,869 @@
+
+        for sample in TimeoutSampler(
+            timeout=600,
+            sleep=10,
+            func=self._check_build_complete,
+        ):
+            if sample:
+                break
+
+        logger.info("CBT lister pod %s is ready", pod_name)
+
+    def _check_build_complete(self):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `_check_build_complete`

(ANN202)